packages/iov-faucets/src/bovfaucet.ts

```diff
@@ -0,0 +1,24 @@
+import axios from "axios";
+
+import { Address, TokenTicker } from "@iov/bcp-types";
+import { Encoding } from "@iov/encoding";
+
+export class BovFaucet {
+  private readonly url: string;
+
+  constructor(url: string) {
+    if (!url.startsWith("https://")) {
+      throw new Error("Expected url to start with https://");
+    }
+    this.url = url;
+  }
+
+  public async open(address: Address, ticker: TokenTicker = "IOV" as TokenTicker): Promise<void> {
+    const body = {
+      address: Encoding.toHex(address),
+      tokens: [{ ticker: ticker }],
+    };
+
+    await axios.post(this.url, body);
```

Comment:
Did you test this with karma (in browser). I mean, it would be great to have a button saying "get free tokens" in a webapp, so usecase makes sense to me.  I had issues with axios in tendermint rpc in browser. Can you add a test? You may need to make a fix like this (or improve/refactor my fix):

https://github.com/iov-one/iov-core/blob/master/packages/iov-tendermint-rpc/src/rpcclient.ts#L33-L55

Reply:
Good point. Added browser tests but will keep them out of here because of https://github.com/iov-one/tools/issues/71